### ISSUE TITLE
Scrolling over a Spoiler does not work when inside a ScrollArea

### ISSUE THREAD

**Issue by jalik** [mantinedev/mantine#7150]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.14.1

### What package has an issue?

@mantine/core

### What framework do you use?

Vite

### In which browsers you can reproduce the issue?

Firefox and Chrome

### Describe the bug

1. place the pointer over a text in a folded `<Spoiler>` that is located in a `<ScrollArea>`
2. try to scroll down/up
3. nothing happens
4. however moving the mouse on an area with no text seems to work

### If possible, include a link to a codesandbox with a minimal reproduction

https://codesandbox.io/p/sandbox/z2pxl4

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by minosss** [comment]
`ScrollArea` needs height

**Comment by rtivital** [comment]
Yes, it is correct, ScrollArea component must have height, see – https://codesandbox.io/p/sandbox/mantine-spoiler-scroll-bug-forked-qf2jxn?file=%2Fsrc%2FApp.js%3A13%2C11

**Comment by jalik** [comment]
@minosss @rtivital My bad, I tried to isolate the problem by simplifying the code too much, leading me to think the issue was with the `ScrollArea`, but in the original code, the `Spoiler` elements are nested in a `Tabs` in a `Modal`, so I don't use a `ScrollArea` directly.

I finally found that **the issue was caused by an overflow CSS rule** that I should have removed, I've added it to set a max height on each tab, but later removed that max height, leaving the overflow rule and a bug...
In case you want to see, I've updated the codesandbox.

In the end it's working fine !

Thank you for your time and sorry for the false alert.
